dolphinscheduler-task-plugin/dolphinscheduler-task-seatunnel/src/main/java/org/apache/dolphinscheduler/plugin/task/seatunnel/SeatunnelTask.java

```diff
@@ -169,9 +170,33 @@ protected List<String> buildOptions() throws Exception {
         String filePath = buildConfigFilePath();
         createConfigFileIfNotExists(scriptContent, filePath);
         args.add(filePath);
+        args.addAll(generateTaskParameters());
         return args;
     }
 
+    private List<String> generateTaskParameters() {
```

Comment:
Please add these feature to the docs.
https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/zh/guide/task/seatunnel.md
https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/guide/task/seatunnel.md